test/Parse/structure_overflow.swift

```diff
@@ -0,0 +1,8 @@
+// RUN: not %target-swift-frontend -typecheck %s 2>&1 | %FileCheck %s
+
+// level 512.
+// CHECK: :[[@LINE+1]]:256: error: structure nesting level exceeded maximum of 256
```

Comment:
Without `Fatal`, this test case emits ton of:
```
error: expected ')' in expression list
error: expected '}' at end of closure
```

AFAIK, fatal error is just a `error` for SourceKit/Xcode.
For instance `no such module` error is `Fatal`.